ocs_ci/ocs/container.py

```diff
@@ -0,0 +1,274 @@
+
+import logging
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.utility import templating
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.exceptions import CommandFailed
+
+from tests.helpers import (create_unique_resource_name,
+                           get_admin_key, get_cephfs_name,
+                           get_cephfs_data_pool_name)
+
+log = logging.getLogger(__name__)
+
+
+class Container(object):
```

Comment:
I think we might want to reconsider the name `Container`. Given the product/technology we are testing this might lead to some confusion.

Reply:
I dont know what else makes a better name :) , I can call it PVCStor as well.